src/test/fs_tests.cpp

```diff
@@ -118,4 +118,28 @@ BOOST_AUTO_TEST_CASE(fsbridge_fstream)
     }
 }
 
-BOOST_AUTO_TEST_SUITE_END()
\ No newline at end of file
+#ifndef WIN32
```

Comment:
Could be narrowed to `__MINGW64__`, but `WIN32` is ok as it is coordinated with a new `feature_dirsymlinks.py` test.

Reply:
WIN32 is "all windows" so I think this is correct. No need to narrow it if we know it's true for all.

Reply:
I mean, this test, being applied on top of the master branch, [passes](https://cirrus-ci.com/task/5367730952994816) for MSVC build.